apps/dictionaries/service.py

```diff
@@ -156,11 +156,10 @@ def on_create(self, docs):
                 pass
 
         for doc in docs:
-            if self.find_one(req=None, name=doc['name'],
-                             language_id=doc['language_id'],
-                             type=doc.get('type', DictionaryType.DICTIONARY.value)):
+            if self.is_valid_dictonary(doc):
```

Comment:
I think that method should be renamed, it's a bit confusing that when `is_valid_dictionary` returns True you raise exception. maybe `is_duplicate_dictionary` or something